Source/utils/find_steam_game.h

```diff
@@ -0,0 +1,262 @@
+/*
```

Comment:
This file should go in the 3rdParty folder since it's not developed/maintained by us.